src/SignalRServiceExtension/Bindings/SignalRInputBindings/SignalRConnectionInputBinding.cs

```diff
@@ -1,38 +1,42 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.Collections.Generic;
+using System.Reflection;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Protocols;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
 
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
-    internal class SignalRConnectionInputBinding : IBinding
+    internal class SignalRConnectionInputBinding : BindingBase<SignalRConnectionInfoAttribute>
```

Comment:
When binding the `SignalRConnectionInfo` to a parameter in non-C# language, Function tries to get the type of your binding data first. It will be `string` by default. Binding need to implement `IBindingRuleProvider` to provide the type of object. Did it work properly now in js?

Reply:
the `IBindingRuleProvider` is implemented in `FilteringBindingProvider`. the `InputBindingProvider` is an inner provider to `FilteringBindingProvider`. We don;t need to care about languages.

Reply:
will support it later